generator/src/test/java/com/scottlogic/deg/generator/decisiontree/DecisionTreeSimplifierTests.java

```diff
@@ -0,0 +1,100 @@
+package com.scottlogic.deg.generator.decisiontree;
+
+import com.scottlogic.deg.generator.Field;
+import com.scottlogic.deg.generator.ProfileFields;
+import com.scottlogic.deg.generator.constraints.atomic.AtomicConstraint;
+import com.scottlogic.deg.generator.constraints.atomic.IsInSetConstraint;
+import com.scottlogic.deg.generator.constraints.atomic.IsNullConstraint;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+class DecisionTreeSimplifierTests {
+    // TODO: Simplifier tests needs fleshing out
```

Comment:
Is this TODO still valid?

Reply:
On discussion with Steve tests have been added to cover the specifics of the cases raised in the issue but not to fully cover the class under test. We will either add the tests here and now or create a task to revisit full testing of the simplifier.

Reply:
new issue #628 created